### PR TITLE
Issue #2791

### PR DESCRIPTION
**Pull request by Hypolite**
Simple SQL fix for request approval notification issue #2791

### CODE REVIEW

File: mod/ping.php [hunk 1]
```diff
@@ -123,11 +123,10 @@ function ping_init(&$a) {
 		$mail = count($mails);
 
 		if ($a->config['register_policy'] == REGISTER_APPROVE && is_site_admin()){
-			$regs = q("SELECT `contact`.`name`, `contact`.`url`, `contact`.`micro`, `register`.`created`, COUNT(*) as `total` FROM `contact` RIGHT JOIN `register` ON `register`.`uid`=`contact`.`uid` WHERE `contact`.`self`=1");
-			if ($regs)
-				$register = $regs[0]['total'];
+			$regs = q("SELECT `contact`.`name`, `contact`.`url`, `contact`.`micro`, `register`.`created` FROM `contact` RIGHT JOIN `register` ON `register`.`uid`=`contact`.`uid` WHERE `contact`.`self`=1");
+			$register = count($regs);
```
Comment (annando) [inline mod/ping.php]: We are having a function that checks the return data from a query:

````
if (dbm::is_result($regs))
````
Although in this case it is not really needed, I would like to have it added for consistency reasons.

Could you add it?

Comment (Hypolite) [inline mod/ping.php]: It's already used in line 254, not sure where or how you want to use it. The result count directly after query is already used in lines 114 and 123, should they be protected by is_result too?

Comment (annando) [inline mod/ping.php]: You are right. This is inconsistent. :-) So I guess I don't have any objections accepting the pull request ;-)

Comment (Hypolite) [inline mod/ping.php]: Yay!

**Comment by tobiasd** [comment]
Just noticed it, for the future, please make your PRs against the `develop` branch.

**Comment by Hypolite** [comment]
Yes, I noticed after the fact.

**Pull request merged**